src/js/views/patients/schedule/schedule-list.scss

```diff
@@ -102,6 +102,12 @@
   width: 63%;
 }
 
+.schedule-list__action-details {
+  color: $black-60;
+  padding: 0;
+  width: 16px;
```

Comment:
So _probably_ the form column should also be a px setting and the action column should be `calc(68% -32px)`.  And the headings would also need to accommodate for that.  I'm not sure what adding 16px is doing without any percentage adjustments, but something somewhere ain't quite right.